tests/test.tracking.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var tracking = require('tracking');
+
+
+describe('tracking uninitialized', function() {
+
+  beforeEach(function() {
+    this.oldTrackingEnabled = tracking.enabled;
+    this.oldTrackingInitialized = tracking.initialized;
+    tracking.enabled = true;
+    tracking.initialized = false;
```

Comment:
So the thought behind this was initially about avoiding the setup piece on every require. A module exporting an instance should implicitly be a singleton (afaict though I've seen some things saying you might get a different object if your path for a require is different) , I just made this more explicit in the code.

The reason for the single instance was to track state for when the setup had been completed. This way we can show errors if setPage / sendEvent gets called but the analytics code hasn't been inited.